agimus_controller/tests/test_robot_models.py

```diff
@@ -0,0 +1,164 @@
+from copy import deepcopy
+from os.path import dirname
+import unittest
+
+import example_robot_data as robex
+import numpy as np
+import pinocchio as pin
+
+from agimus_controller.factory.robot_model import RobotModelParameters, RobotModels
+
+
+class TestRobotModelParameters(unittest.TestCase):
+    def test_valid_initialization(self):
+        """Test that the dataclass initializes correctly with valid input."""
+
+        robot = robex.load("panda")
+        urdf_path = robot.urdf
+        srdf_path = robot.urdf.replace("urdf", "srdf")
+        urdf_meshes_dir = dirname(dirname(robot.urdf))
```

Comment:
```suggestion
        urdf_path = Path(robot.urdf)
        srdf_path = Path(robot.urdf.replace("urdf", "srdf"))
        urdf_meshes_dir = robot.urdf.parent.parent
```
Won't this work the same?